README.md

```diff
@@ -1,13 +1,24 @@
-Beautiful text chat for your community and website.
+## Scrollback, where communities hang out
 
-# Installation
+Scrollback gives you beautiful text chat for your community and website. [Try it out](https://scrollback.io/me)! 
 
-Run this command in terminal:
+You can even [embed Scrollback room](https://github.com/scrollback/scrollback/wiki/Basic-Usage#embed-scrollback-room) on your webpage/blog.
 
-    wget https://raw.githubusercontent.com/scrollback/scrollback/master/tools/install.sh -O install.sh && chmod +x install.sh && ./install.sh
+## Install
 
-To run Scrollback run:
+Scrollback is hosted Free Software but if you wanted to run it on your computer, you can! To install Scrollback locally:
 
-	sudo npm start
+1. Run `wget https://raw.githubusercontent.com/scrollback/scrollback/master/tools/install.sh -O install.sh && chmod +x install.sh && ./install.sh`.
+1.Start Scrollback with `sudo npm start`.
```

Comment:
This is installation instructions w.r.t to old install.sh. I didn't know how long it would take before I'd finish the re-write so the aim was to make existing README.md more helpful.